src/Storefront/Resources/app/storefront/src/plugin/forms/form-submit-loader.plugin.js

```diff
@@ -108,8 +113,16 @@ export default class FormSubmitLoaderPlugin extends Plugin {
 
             const loader = new ButtonLoadingIndicator(submitButton, this.options.indicatorPosition);
             loader.create();
+            this._submittedButtonLoaders.push(loader);
         });
 
         this.$emitter.publish('beforeSubmit');
     }
+
+    /**
+     * Will remove existing loading indicators from submit buttons
+     */
+    removeLoadingIndicator() {
+        this._submittedButtonLoaders.forEach(loader => loader.remove());
+    }
```

Comment:
I think we need to reset the array here as well. Otherwise more and more loaders are added when I submit multiple times (with errors)

```js
this._submittedButtonLoaders = [];
```